cfn-resources/alert-configuration/template.yml

```diff
@@ -25,5 +25,5 @@ Resources:
         Variables:
           MODE: Test
           LOG_LEVEL: debug
-          MONGODB_ATLAS_OPS_MANAGER_URL:
+          MONGODB_ATLAS_BASE_URL:
```

Comment:
👍 Also, lets make sure to test for the impact of changing the URL env variable, in case there are old occurances